### PR TITLE
Replaced hardcoded installdirs with GNUInstallDirs. Fixed some status msgs

### PR DESCRIPTION
**Pull request by ethouris**

### CODE REVIEW

**Comment by ethouris** [comment]
As per #313

File: CMakeLists.txt [hunk 1]
```diff
@@ -523,20 +531,21 @@ endif()
 
 # Cygwin installs the *.dll libraries in bin directory and uses PATH.
 
-set (INSTALL_SHARED_DIR lib)
+set (INSTALL_SHARED_DIR ${CMAKE_INSTALL_LIBDIR})
 if (CYGWIN)
-	set (INSTALL_SHARED_DIR bin)
+	set (INSTALL_SHARED_DIR ${CMAKE_INSTALL_BINDIR})
```
Comment (ocrete) [inline CMakeLists.txt]: Isn't this not set if you dont use GNUInstalldirs?

File: CMakeLists.txt [hunk 2]
```diff
@@ -26,6 +26,21 @@ set (CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/scripts")
...
+set_if(LINUX       ${CMAKE_SYSTEM_NAME} MATCHES "Linux")
+set_if(MICROSOFT   WIN32 AND (NOT MINGW AND NOT CYGWIN))
+set_if(SYMLINKABLE LINUX OR DARWIN OR CYGWIN)
+
+# Not sure what to do in case of compiling by MSVC.
+# This will make installdir in C:\Program Files\SRT then
+# inside "bin" and "lib64" directories. At least this maintains
+# the current status. Shall this be not desired, override values
+# of CMAKE_INSTALL_BINDIR, CMAKE_INSTALL_LIBDIR and CMAKE_INSTALL_INCLUDEDIR.
+if (NOT DEFINED CMAKE_INSTALL_LIBDIR)
```
Comment (ocrete) [inline CMakeLists.txt]: You may want to just disable it on Windows and enable everywhere else? You want to also replace include/ etc

Comment (ethouris) [reply CMakeLists.txt]: I was thinking about it, but actually I have never seen any "installing standards" on Windows, especially as it comes to libraries, and GNUInstallDirs works on Windows as well. AFAIK cmake on Windows allows it to install in `C:\Program Files` or kinda like that, followed by the project name and then inside this directory the GNU directory layout is as good as any other. I've moved these platform dependent variables just for that purpose, but then I finally came to the conclusion as described in the above comment.

File: CMakeLists.txt (hunk 1 above)
Comment (ethouris) [reply CMakeLists.txt]: But I do use GNUInstallDirs, unless CMAKE_INSTALL_LIBDIR is set. I state all others are set then as well.

**Pull request merged**